packages/browser-tests/src/server.ts

```diff
@@ -1,25 +1,57 @@
-import { ChildProcess, exec } from "child_process";
-import * as net from "net";
-import { dirname, join } from "path";
 import { fileURLToPath } from "url";
+import * as path from "path";
+import * as fs from "fs";
 
 import { chromium } from "@playwright/test";
 import cors from "cors";
 import express, { Request, Response } from "express";
+import * as net from "net";
 
 import adminRouter from "./routes/admin.js";
 import { setPage, getPage, closeBrowser } from "./browser/index.js";
 
-const __filename = fileURLToPath(import.meta.url);
-const __dirname = dirname(__filename);
 
 const app = express();
 
 app.use(cors());
 app.use(express.json());
+
+// Serve built shared API for browser import
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const distRoot = path.resolve(__dirname, "..");
+const sharedDir = path.resolve(distRoot, "shared");
+// Serve bootstrap script from source (not emitted by tsc)
+const assetsDir = path.resolve(distRoot, "..", "src", "assets");
+app.use("/shared", express.static(sharedDir));
+app.use(express.static(assetsDir));
+
+// Resolve installed @waku/sdk version for CDN import map
+let wakuCdnVersion = "0.0.30";
+try {
+  const pkgPath = path.resolve(distRoot, "..", "package.json");
+  const pkgRaw = fs.readFileSync(pkgPath, "utf-8");
+  const pkg = JSON.parse(pkgRaw);
+  const declared = (pkg.dependencies?.["@waku/sdk"] || pkg.devDependencies?.["@waku/sdk"]) as string | undefined;
+  if (declared) {
+    wakuCdnVersion = String(declared).replace(/^[^0-9]*/, "");
+  }
+} catch (e) {
+  // ignore resolution errors; fallback version is used
+}
+// Parameterize CDN base, defaults, and stub values via env
+const wakuCdnBase = process.env.HEADLESS_WAKU_CDN_BASE || "https://esm.sh";
+const wakuCdnUrl = `${wakuCdnBase}/@waku/sdk@${wakuCdnVersion}?bundle`;
+const defaultClusterId = process.env.HEADLESS_DEFAULT_CLUSTER_ID
+  ? parseInt(process.env.HEADLESS_DEFAULT_CLUSTER_ID, 10)
+  : 42;
+const defaultShard = process.env.HEADLESS_DEFAULT_SHARD
+  ? parseInt(process.env.HEADLESS_DEFAULT_SHARD, 10)
+  : 0;
+const stubPeerId = process.env.HEADLESS_STUB_PEER_ID || "mock-peer-id";
 app.use(adminRouter);
 
-let headlessServerProcess: ChildProcess | undefined;
+// Removed: external static server no longer required
```

Comment:
what is the purpose of this comment line? it just mentions something was removed from before, it has zero value